src/PhpImap/Imap.php

```diff
@@ -911,10 +911,10 @@ public static function sort(
         \imap_errors(); // flush errors
 
         $imap_stream = self::EnsureConnection($imap_stream, __METHOD__, 1);
-        $reverse = $reverse;
```

Comment:
`php-cs-fixer` complains about the missing `use const PHP_MAJOR_VERSION;` statement at the top.

Please add this by running `./vendor/bin/php-cs-fixer fix --allow-risky=yes --no-interaction -v`.